code/modules/antagonists/wraith/critters/abilities/plaguerat.dm

```diff
@@ -149,6 +149,9 @@ ABSTRACT_TYPE(/datum/targetable/critter/plague_rat)
 		if (BOUNDS_DIST(holder.owner, target) > 0)
 			boutput(holder.owner, "<span class='alert'>That is too far away to bite.</span>")
 			return TRUE
+		if (holder.owner.hasStatus("disposals")) //No biting in Disposals, that's cheating.
+			boutput(holder.owner, "<span class='alert'>You definitely can't bite through pipes.</span>")
+			return FALSE
```

Comment:
```suggestion
			return TRUE
```
abilities return `TRUE` for a failed cast, don't ask why

Reply:
I should really stop coding right before I go to sleep